qbft/state.go

```diff
@@ -28,22 +24,22 @@ type IConfig interface {
 	GetNetwork() Network
 	// GetTimer returns round timer
 	GetTimer() Timer
+	// GetSignatureVerifier returns the signature verifier for operator signatures
+	GetSignatureVerifier() types.SignatureVerifier
+	// GetCutOffRound returns the round that stops the instance
+	GetCutOffRound() Round
```

Comment:
If `Timer` is in `Config` this can also be in Config.
This isn't in interface.

Besides all those protocol configurations should be well defined in spec because implementations are not supposed to pick whatever values they want, else they would deviate.

Anyhow currently https://github.com/ssvlabs/ssv-spec/blob/da7262a6acca57fb6ac1fd3b05d617c8f2f4098c/qbft/timeout.go#L17-L21 here are the configuration values.

There are PRs in place to properly spec the timer that kept on losing priority